easybuild/easyconfigs/m/matplotlib/matplotlib-2.0.2-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('freetype', '2.7.1'),
```

Comment:
If you're only using `freetype` easyconfigs from EasyBuild, you should not be seeing this at all...

Which modules do you have available for `freetype` `2.7.1`? And which `libpng` version are they using as dependency?

To get this PR merged, you should either change the `libpng` version to `1.6.28`, or change the `freetype` dependency to use the one that uses `libpng` `1.6.29`.

Whether you should also rebuild `matplotlib` with that updated easyconfig is up to you, but if you don't you may run into a similar issue again later.